docs/CMakeLists.txt

```diff
@@ -29,8 +29,8 @@ if(NOT ${projectPrefix}BUILD_DOCS)
     return()
 endif()
 
-# set paths to Conan packages (needed for doxygen)
-# TODO: remove when Conan will learn how to properly create imported build tools targets
+# set paths to Conan packages (needed for doxygen) TODO: remove when Conan will learn how to properly create imported
+# build tools targets
```

Comment:
This shows why I disable comment formatting. `cmake-format` does not play well with [Semantic Line Breaks](https://sembr.org/).

Reply:
Then, do we want to disable comment formatting in general or put fence here as well?

Reply:
Disable it ;-)